examples/utility.py

```diff
@@ -0,0 +1,32 @@
+import math
+
+import cv2
+import depthai as dai
+import numpy as np
+
+
+def calculateDispScaleFactor(device):
+    calib = device.readCalibration()
+    eeprom = calib.getEepromData()
+    leftCam = calib.getStereoLeftCameraId()
+    if leftCam != dai.CameraBoardSocket.AUTO:
+        camInfo = eeprom.cameraData[leftCam]
+        baseline = abs(camInfo.extrinsics.specTranslation.x * 10)  # cm -> mm
+        fov = calib.getFov(calib.getStereoLeftCameraId())
+        focal = (camInfo.width / 2) / (2. * math.tan(math.radians(fov / 2)))
+    else:
+        print("Warning: calibration data missing, using OAK-D defaults")
+        baseline = 75
+        fov = 71.86
+        focal = 440
+    return baseline * focal
+
+
+def parseDepthPacket(inPacket, dispScaleFactor, maxDisparity, colorMap=cv2.COLORMAP_JET):
```

Comment:
I added the subpixel handler now. Since we want the python examples to stay "low level", I would stick with functions. If someone would like to use class-based approach, I would recommend using SDK